eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Examples/Consensus.hs

```diff
@@ -442,17 +442,17 @@ exampleTxBodyShelley =
     auxiliaryDataHash =
       AuxiliaryDataHash $ mkDummySafeHash (Proxy @StandardCrypto) 30
 
-exampleMetadataMap :: Map Word64 Metadatum
-exampleMetadataMap =
+exampleAuxDataMap :: Map Word64 Metadatum
```

Comment:
Wondering why the rename, since the type is  still `Metadatum` ?

Reply:
Another good question. `Metadatum` is one small piece of `ShelleyTxAuxData`. Maybe we could rename it too, but I suggest we do it in a separate PR, since it is only an implementation detail of the actual type that needed renaming.

Reply:
Because it is used to make a TxAuxData value?

Reply:
Makes sense!  My check was sort of mechanical, but of course there's no point to have the type of the variable encoded in the name  :sweat_smile:    :+1: